.circleci/config.yml

```diff
@@ -42,81 +42,81 @@ workflows:
 
```

Comment:
I think the whole pushing to catalogs/collections part can be removed. We want to use it as a binary in the CLI mode only.